tests/__init__.py

```diff
@@ -0,0 +1,301 @@
+"""
+(C) Copyright 2013-2021 NuoDB, Inc.  All Rights Reserved.
+
+This software is licensed under a BSD 3-Clause License.
+See the LICENSE file provided with this software.
+"""
+
+import os
+import string
+import random
+import socket
+import tempfile
+import logging
+import time
+import copy
+import shutil
+import errno
+
+import subprocess
+import json
+
+DB_OPTIONS = []
+
+DATABASE_NAME = 'pynuodb_test'
+DBA_USER      = 'dba'
+DBA_PASSWORD  = 'dba_password'
+
+ap_conn = None
+ar_path = None
+db_created = False
+db_started = False
+
+
+def waitforsql(tmout):
+    import pynuodb
+    host = 'localhost'
+    port = os.environ.get('NUODB_PORT')
+    if port:
+        host += ':' + port
```

Comment:
This method of determining the agent address is not great: ideally we'd use the same server we use to start/stop things and retrieve the proper address and port from that.  This info is available but it's not easy to get.